dataactcore/README.md

```diff
@@ -64,19 +60,17 @@ Note that all new ORM objects must inherit from the `declarative_base` object an
 
 Additional fields exist on some of the models to enable the automatic population of foreign key relationships. These fields use the `relationship` function to signify a mapping to another table.  More information on SQLAlchemy ORM objects can be found on the [official website](http://docs.sqlalchemy.org/en/latest/orm/tutorial.html#create-a-schema).
 
-Database interfaces are defined for each database used within the project. Each interface inherits base functionality from `BaseInterface` and defines both the database name and credentials file location. Where required, interfaces are extended in the other packages to add additional functionality.
+Database interfaces are defined for logical data functions in the data broker (e.g., validations). Each interface inherits base functionality from `BaseInterface`, which defines the location of connection credentials and is responsible for overall session and connection handling. Where required, interfaces are extended to add additional functionality.
 
 #### Scripts
 
-The `scripts/` folder contains various python scripts used in the DATA Act broker backend install process, including database creation scripts. When run directly, the following scripts take no parameters and stand up all required tables within each database:
+The `scripts/` folder contains various python scripts used in the DATA Act broker backend install process, including the scripts that create the broker's backend database and handle the initial data population:
 
 - setupJobTrackerDB (Creates job_tracker database)
 - setupErrorDB      (Creates the error database)
 - setupUserDB       (Creates the user database)
 - setupAllDB        (Creates all of the needed databases)
```

Comment:
Should we include setupValidationDB and setupJobQueueDB in this list?  It might also make sense to remove this list as there are several other scripts not mentioned here.